documentation/general/design/dash-holistic-view.md

```diff
@@ -0,0 +1,617 @@
+  - [Switch Abstraction Interface (SAI) DASH extension](#switch-abstraction-interface-sai-dash-extension)
+  - [ASIC Drivers](#asic-drivers)
+  - [DASH capable ASICs](#dash-capable-asics)
+- [SONiC integration](#sonic-integration)
+  - [DASH single DPU on NIC](#dash-single-dpu-on-nic)
+  - [DASH appliance](#dash-appliance)
+    - [High Level Architecture](#high-level-architecture)
+    - [Low level architecture](#low-level-architecture)
+  - [DASH smart switch](#dash-smart-switch)
+    - [High level architecture](#high-level-architecture-1)
+    - [Low level architecture](#low-level-architecture-1)
+- [Example of network representation](#example-of-network-representation)
+  - [Deployment](#deployment)
+- [API](#api)
+  - [Switch Abstraction Interface (SAI) API](#switch-abstraction-interface-sai-api)
+  - [DASH pipeline API](#dash-pipeline-api)
+    - [Behavioral model](#behavioral-model)
+  - [Functional testing](#functional-testing)
+  - [Conformance and performance testing](#conformance-and-performance-testing)
+- [Appendix](#appendix)
+  - [A day in the life of a DASH packet](#a-day-in-the-life-of-a-dash-packet)
+  - [A day in the life of a DASH SDN controller](#a-day-in-the-life-of-a-dash-sdn-controller)
+  - [A day in the life of a DASH container](#a-day-in-the-life-of-a-dash-container)
+- [References](#references)
+
+## Introduction
+
+This document introduces the **Disaggregated API for SONiC Hosts** (**DASH** for
+short) and provides an overview of the project, the supporting elements and its
+architecture.
+
+![dash-words-cloud](./images/dash-words-cloud.png)
+
+DASH is an open source project whose goal is to deliver a **behavioral model**
+and related **test cases**. Technology providers must conform to the model and
+test cases when designing and building network devices. These devices are meant
+to provide **high-speed network connectivity** to critical cloud enterprise
+applications. 
+
+DASH extends SONiC APIs and a related comprehensive set of object models that
+describe networking services for the cloud. The project enlists cloud and
+enterprise technology providers to collaborate and further extend DASH to meet
+their specific needs.
+
+### Why DASH? 
+
+The current technologies in the market by necessity provide general purpose APIs
+to perform general purpose actions (ACLs, forwarding tables, metering, etc.).
+The reason is that it is impossiible to target each specific technology
+provider's implemantation. Obviusely, a general purpose approach can just do so
+much and works against high performance connectivity. 
+
+DASH takes a different approach. It does not make any assumption or enforce any
+implementation details, such as programming language. It delegates to the
+technology providers such decisions so they can best employ their know-how and
+competitive edge to obtain the best possible results, as long as the agreed upon
+DASH behaviroal model and test cases are satisfied. 
```

Comment:
```suggestion
Because it has been difficult/impossible to target each specific technology provider's implementation(s), the technologies in the market currently provide general purpose APIs to perform general purpose actions (such as ACLs, forwarding tables, metering, etc...).  Again, these are not targeted to specific cloud/enterprise usage.  Obviously, a general purpose approach delivers only so much, and may work against high performance connectivity. 

DASH takes a different approach. DASH is publicizing Behavioral Models to optimize exactly 'what' clouds/enterprises are doing, to optimize against their services. This is the first time 'what we do' has been exposed.  It does not make assumptions or enforce implementation details, such as a programming language. It delegates these decisions to technology providers in order to best employ their expertise and competitive edge to obtain the best possible results (*given that the agreed upon
DASH behavioral model and test cases are satisfied*).
```